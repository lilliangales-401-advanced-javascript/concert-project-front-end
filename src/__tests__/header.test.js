import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../components/header/header';

describe('<Form />', () => {
  it('Form is rendered on the DOM', () => {
  // eslint-disable-next-line
    const app = shallow(<Header />);
    expect(app.find('h1').exists()).toBe(true);
  });
  it('renders correctly', () => {
    const tree = renderer
      .create(<Header page="http://www.facebook.com">Facebook</Header>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
