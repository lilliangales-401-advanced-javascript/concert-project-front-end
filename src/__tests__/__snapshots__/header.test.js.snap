// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> renders correctly 1`] = `
<header>
  <h1>
    Headers R Fun
  </h1>
</header>
`;
